File: leoz-node/src/main/java/org/deku/leoz/node/data/repositories/master/SectorRepository.java
package org.deku.leoz.node.data.repositories.master;

import org.deku.leoz.node.data.entities.MstSector;
import org.deku.leoz.node.data.entities.MstSectorId;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.querydsl.QueryDslPredicateExecutor;

/**
 * Created by masc on 16.05.15.
 */
public interface SectorRepository extends JpaRepository<MstSector, MstSectorId>, QueryDslPredicateExecutor<MstSector> {
}

File: leoz-node/src/main/java/org/deku/leoz/node/data/repositories/master/SectorRepository.kt
package org.deku.leoz.node.data.repositories.master

import org.deku.leoz.node.data.jpa.MstSector
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.querydsl.QueryDslPredicateExecutor

/**
 * Created by masc on 16.05.15.
 */
interface SectorRepository :
        JpaRepository<MstSector, Long>,
        QueryDslPredicateExecutor<MstSector>
